src/Adapter/LocalStorage.php

```diff
@@ -197,6 +197,14 @@ public function getURL($path)
 
         return $this->webUrl.'/'.$path;
     }
+    
+    /**
+     * {@inheritDoc}
+     */
+    public function getExpiringURL($path, $expiresAt) {
+        // Local storage cannot provide expiring URLs. Falling back...
```

Comment:
Perhaps we could just implement the feature for local storage? Something like having a cached table of file names and expiry times and we could run a garbage collector on it now and then.

I mean, it's not ideal, but if we're treating this as a fallback, it would work as a fallback. And having a feature generally supported everywhere in the library would be kind of cool, I'd be happy to knock something like this together.